ingestion/src/metadata/ingestion/source/dashboard/superset/db_source.py

```diff
@@ -87,7 +86,7 @@ def get_column_list(self, table_name: str) -> Iterable[FetchChart]:
         try:
             if table_name:
                 col_list = self.engine.execute(
-                    sql.text(FETCH_COLUMN), table_name=table_name.lower()
```

Comment:
@ulixius9 have you tried with `text` using `from sqlalchemy import text`?

Reply:
Hey @TeddyCr, I just tested it out and still it does not work, it throws the syntax error as it converts the `%` into `%%(table_name)s` to escape it and then the text binding does not work.

Hence I had to remove the text wrapping and pass the string as it is